### PR TITLE
API addition for translations

### PR DESCRIPTION
**Pull request by PrimordialMoros**
Adds a method to check if a translation key has any registered translations.

This allows one to easily handle a missing translation key using a custom fallback.

### CODE REVIEW

**Comment by zml2008** [comment]
It's not possible to know serverside whether or not a translation exists - the translatable component supports any key defined in a client resource pack as well.

**Comment by PrimordialMoros** [comment]
I was thinking about plugin registered keys that are unlikely to exist on client resource packs. Perhaps, should mention this limitation in the documentation, or change the method name to better reflect that?

File: api/src/main/java/net/kyori/adventure/translation/TranslationRegistry.java [hunk 1]
```diff
@@ -71,6 +71,15 @@
...
   }
 
+  /**
+   * Checks if any translations have been registered for the specified key.
+   *
+   * @param key a translation key
+   * @return whether the key has any registered translations
+   * @since 4.3.1
+   */
+  boolean hasTranslation(final @NonNull String key);
```
Comment (kashike) [inline api/src/main/java/net/kyori/adventure/translation/TranslationRegistry.java:81]: `contains`?

Comment (PrimordialMoros) [reply api/src/main/java/net/kyori/adventure/translation/TranslationRegistry.java]: Updated it

**Review by zml2008 — APPROVED** [review]

**Pull request merged**